### DISCUSSION TITLE
4.6.23，直播讨论区经常漏弹幕，特别没有勋章的

### DISCUSSION THREAD

**Discussion by xiaoyanxyxy** [ywmoyue/biliuwp-lite#433] [bug]
没有粉丝勋章的用户 在讨论区发送弹幕 经常漏掉 不显示  一些弹幕也会漏掉

**Comment by xiaoyanxyxy** [comment]
直播里是特别过滤掉没有勋章的粉丝讨论和评论么？ 漏掉好多互动啊

**Comment by GD-Slime** [comment]
~确实有此情况。无粉丝牌的弹幕不会显示。明天详细排查一下~
已修 #435 等合并发新版本 @xiaoyanxyxy 或者可以自己编译dev

**Reply by GD-Slime** [reply]
如果还有其他佩戴粉丝牌但是还是不显示的麻烦测试一下发出来，一并修了
